src/main/java/com/aws/greengrass/clientdevices/auth/certificate/CertificateStore.java

```diff
@@ -83,6 +83,7 @@ public CertificateStore(Path workPath) {
         this.workPath = workPath;
     }
 
+
```

Comment:
nit: remove

Reply:
I'm also going to nitpick on this. Please remove